modules/wkt/test/wkb/parse-wkb.spec.ts

```diff
@@ -1,45 +1,50 @@
 import test from 'tape-promise/tape';
 import {fetchFile} from '@loaders.gl/core';
+import {isWKB} from '../../src/lib/parse-wkb-header';
 import parseWKB from '../../src/lib/parse-wkb';
 import {parseTestCases} from './utils';
 
 const WKB_2D_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdata2d.json';
 const WKB_Z_TEST_CASES = '@loaders.gl/wkt/test/data/wkb-testdataZ.json';
 
-test('parseWKB2D', async (t) => {
+test('parseWKB#2D', async (t) => {
   const response = await fetchFile(WKB_2D_TEST_CASES);
   const TEST_CASES = parseTestCases(await response.json());
 
   // TODO parseWKB outputs TypedArrays; testCase contains regular arrays
   for (const testCase of Object.values(TEST_CASES)) {
     // Little endian
     if (testCase.wkb && testCase.binary) {
-      t.deepEqual(parseWKB(testCase.wkb), testCase.binary);
+      t.ok(isWKB(testCase.wkb), 'isWKB(2D)');
```

Comment:
The test case object also has ewkb, ewkb big endian, etc. We should test those as well.
https://github.com/visgl/loaders.gl/blob/0f28bee0c50f5ed8afa28c8326ab65d7ba7e42b0/modules/wkt/test/wkb/utils.ts#L3-L34